Makefile

```diff
@@ -38,6 +38,23 @@ build_python:
 install-piptools:
 	pip install -U pip-tools
 
+.PHONY: doc_gen_deps # these dependencies are required by protoc gen doc for the protos which have external library dependencies. 
+# which includes grpc-gateway, googleapis, k8.io/api and apimachinery, protocolbuffers 
+doc_gen_deps:
+	rm -rf tmp/doc_gen_deps tmp/protocolbuffers tmp/grpc-gateway tmp/k8s.io
```

Comment:
can we move this to a script?

Reply:
Yes taking this followup where i am fixing the link  which are refrenced from other modules